aws-iot-mitigationaction/aws-iot-mitigationaction.json

```diff
@@ -1,239 +1,248 @@
 {
```

Comment:
JSON files are showing big delta because of fixing indentation using pre-commit hook.